r/R/install-arrow.R

```diff
@@ -98,7 +98,10 @@ OR_SEE_DEV_GUIDE <- paste0(
 SEE_ARROW_INSTALL <- paste(
   "See the Apache Arrow project installation page",
   "<https://arrow.apache.org/install/>",
-  "for how to install the C++ package from a PPA."
+  "to find pre-compiled binary packages for some common Linux distributions,",
+  "including Debian, Ubuntu, and CentOS. You'll need to install",
+  "'libparquet-dev' on Debian and Ubuntu, or 'parquet-devel' on CentOS. This",
+  "will also automatically install the arrow C++ library as a dependency."
```

Comment:
Could you use "Arrow" instead of "arrow"?

Reply:
Done.